bundles/binding/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/common/PckGenerator.java

```diff
@@ -256,6 +256,29 @@ public static String varAbs(LcnDefs.Var var, int value) throws IllegalArgumentEx
     }
 
     /**
+     * Generates a command that send variable status updates.
+     * PCHK provides this variables by itself on selected segments
+     * is only possible with group 4
+     *
+     * @param var the target variable to set
+     * @param value the absolute value to set
+     * @return the PCK command (without address header) as text
+     * @throws IllegalArgumentException
+     */
+    public static String updateStatusVar(LcnDefs.Var var, int value) throws IllegalArgumentException {
+        int id = LcnDefs.Var.toVarId(var);
+        if (id != -1) {
+            // define variable to set, offset 0x01000000
+            int x2cmd = id |= 0x40;
```

Comment:
It doesn't affect the result, but it would be clearer to use the simple bitwise OR operator (`|`) instead of the bitwise inclusive OR and assignment operator (`|=`).